hat/webpack.dev.js

```diff
@@ -5,7 +5,17 @@ const BundleTracker = require('webpack-bundle-tracker');
 // remember to switch in webpack.prod.js and
 // django settings as well
 const LOCALE = 'fr';
-const WEBPACK_URL = 'http://localhost:3000';
+
+// If you launch the dev server with `WEBPACK_HOST=192.168.1.XXX  npm run dev`
+// where 192.168.1.XXX is your local IP address, you can access the dev server
+// from another device on the same network, typically from a mobile device or tablet
+if (process.env.WEBPACK_HOST !== undefined) {
+    WEBPACK_HOST = process.env.WEBPACK_HOST;
+} else {
+    WEBPACK_HOST = 'localhost';
+}
+
+const WEBPACK_URL = 'http://'+ WEBPACK_HOST +':3000';
```

Comment:
It could be removed honestly ... The need is if you want to access a development version from another computer than the one running the webpack dev server (for example if you want to test on a tablet or iphone on the same local network) .. But OK it could be removed ...